master-firmware/src/base/encoder.c

```diff
@@ -70,7 +63,7 @@ static THD_FUNCTION(encoders_thd, arg)
     setup_timer(STM32_TIM3);
 
     uint32_t left_old, right_old;
-    encoders_msg_t values = {0, 0};
+    EncodersPosition values = EncodersPosition_init_zero;
```

Comment:
magic

Reply:
wait until you see support for default values 👍